README.md

```diff
@@ -90,6 +90,38 @@ const PermissionExpandableRow = ({ dependencies: { experimentalFeatures } }) =>
 To make use of this feature by console it can be enabled like this:
 `localStorage.setItem('dopplerExperimental', JSON.stringify({PermissionHistory: true}));`
 
+## Guide Lines
```

Comment:
I think it would be better for this section to be at the beginning of the readme. So people read this first, and then the details.